### ISSUE TITLE
VectorGraphics are not aligned correctly

### ISSUE THREAD

**Issue by xeladotbe** [alexa/apl-viewhost-web#6]
Currently, the viewhost does not take the align property for a VectorGraphic into account.

The APL Core library actually calculates the height, width, X and Y coordinates for a vector graphic and stores the result in the kPropertyMediaBounds property.
https://github.com/alexa/apl-core-library/blob/ba0df9c9f3746800628adb9773e22b206a740f29/aplcore/src/component/vectorgraphiccomponent.cpp#L254

The alignment problem seems to be solved when using the kPropertyMediaBounds property instead of kPropertyInnerBounds.

https://github.com/alexa/apl-viewhost-web/blob/master/js/apl-html/src/components/avg/VectorGraphic.ts#L103

```
- const innerBounds = this.component.getCalculatedByKey<APL.Rect>(PropertyKey.kPropertyInnerBounds);
+ const innerBounds = this.component.getCalculatedByKey<APL.Rect>(PropertyKey.kPropertyMediaBounds);

```

**Comment by tyliec** [comment]
Thank you so much for reporting this issue and providing the sample code for the fix as well. We're looking into fixing this now. ❤️

**Comment by tyliec** [comment]
This has been fixed in the latest release (1.7). Please let us know if this issue occurs again.
